mpc_prediction_topic: car1_30m/mpc_prediction
mpc_metadata_topic: car1_30m/mpc_metadata
reach_pub_topic: hylaa_reach
graph_predictions: True
display_type: NONE #NONE IMAGE VIDEO
display_filename: /home/nvidia/quickzono.png
output_verbosity: NONE
horizon: 1

state_uncertainty:
  px: .1
  py: .1
  psi: .01

input_uncertainty:
  velocity: .1
  delta: .01

